### DISCUSSION TITLE
Integration of Main Pokemon into Reviewer Screen !

### DISCUSSION THREAD

**Discussion by Unlucky-Life** [Unlucky-Life/ankimon#36] [Announcements]
![image](https://github.com/Unlucky-Life/ankimon/assets/77027147/ad1f1073-edff-4912-89f2-03d2a6fb4c6b)
![image](https://github.com/Unlucky-Life/ankimon/assets/77027147/1927702f-8cab-4f0d-9bbe-b9e855ad06dd)
What do you think of the possability to view your mainpokemon in the reviewer at the same time ?

**Comment by Unlucky-Life** [comment]
![image](https://github.com/Unlucky-Life/ankimon/assets/77027147/91b5fe74-f53c-4305-99ae-a9b9946194b5)

**Reply by nurkittel** [reply]
That looks nice and clean. Is it possible to add a field to enable / disable that view option in the Ankimon Menu? That way everyone can decide. 
If you plan to implement damage for your own pokemon in the future that view is perfect as well.

**Reply by Unlucky-Life** [reply]
Yes its in the ankimon configurations/options in the addon overview in anki where you can enable and disable this feature

The damage feature will be included for the next addon

**Reply by shigeyukey** [reply]
That's cool! :-)
I think if the enemy pokemon is in the upper right corner, it would look like a battle, but it might overlap with Anki's flag.

**Reply by Unlucky-Life** [reply]
I see what i can do but that sounds good !

**Reply by Unlucky-Life** [reply]
![image](https://github.com/Unlucky-Life/ankimon/assets/77027147/1927702f-8cab-4f0d-9bbe-b9e855ad06dd)=> slight redesign, what do you think ?

**Reply by shigeyukey** [reply]
That's great! I think the XP bar added is also interesting idea.

**Reply by nurkittel** [reply]
To be honest I would choose the other flat option since it looks cleaner. But I guess it depends. Like Anki itself, I would just make these settings customizable, that way everyone can decide which design fits. 

Very slim xp bar and reduced thickness of the hp bars slightly would be an option I guess. Since quite some cards expand vertical try to take as little as possible of the total review screen. My two cents....
